### ISSUE TITLE
Can't build on ios@4.0.1

### ISSUE THREAD

**Issue by zwacky** [Microsoft/cordova-plugin-code-push#56]
requirements:
```
cordova platform add ios@4.0.1
```

i get the following build error:

*No visible @interface for 'UIView' declares the selector 'loadRequest:'*
```
- (void)loadURL:(NSURL*)url {
    [((CDVViewController *)self.viewController).webView loadRequest:[NSURLRequest requestWithURL:url]];
}
```

**Comment by BrunoFDeveloper** [comment]
+1

**Comment by dlebu** [comment]
@zwacky @BrunoFDeveloper We will release an update today fixing this bug. Thanks for reporting it!

**Comment by zwacky** [comment]
the commit you referenced did fix this, I tested it :+1:

**Comment by dlebu** [comment]
We just released version 1.5.0-beta to npm. The release includes a fix for this issue. Thanks again!

**Comment by BrunoFDeveloper** [comment]
@dlebu and @zwacky Thank you !! now fixed, re-do magic!
